infoTree/getURLfile.js

```diff
@@ -0,0 +1,18 @@
+function getURLindex(label){
+    //first find which URL maps to it
+    let targetURL = "";
+    for (int i = 0; i < whereDoesItGo.length; ++i){
+        let cur = whereDoesItGo[i];
+        if (cur.label === label)
+            targetURL = cur.value;
+    }
+    //found mapped URL, now check in URLS.js array to find which file to use
+    let URLindex;
+    for (int URLindex = 0; URLindex < leafURLS.length; ++URLindex){
```

Comment:
leafURLS also not defined

Reply:
Also global

Reply:
Fixed using import in most recent update